### ISSUE TITLE
Missed OpCapability

### ISSUE THREAD

**Issue by iliya-diyachkov** [KhronosGroup/LLVM-SPIRV-Backend#7]
There is missed OpCapability  in 10 tests:
OpCapability Int64Atomics in capability-Int64Atomics.ll, in capability-Int64Atomics-store.ll
OpCapability Int8 in capability-integers.ll
OpCapability Float16Buffer in half_extension.ll
OpCapability SampledBuffer in image_dim.ll 
OpCapability DeviceEnqueue in spirv.Queue.ll
OpCapability SubgroupDispatch in transcoding/ReqdSubgroupSize.ll
OpCapability SampledBuffer in transcoding/cl-types.ll
OpCapability ImageBasic in transcoding/image_with_access_qualifiers.ll
OpCapability Float64 in transcoding/optional-core-features-multiple.ll

**Comment by iliya-diyachkov** [comment]
OpCapability Int64Atomics was supported in bc0a7d794c83125cbca82fec56003722d1cfeeea.

Fails with OpCapability Int8 and OpCapability Float64 are caused by the missed support for global variables unreferenced in function code.

**Comment by iliya-diyachkov** [comment]
Now only these two tests still fail:
```
OpCapability SubgroupDispatch in transcoding/ReqdSubgroupSize.ll
OpCapability SampledBuffer in transcoding/cl-types.ll
```

**Comment by iliya-diyachkov** [comment]
transcoding/cl-types.ll was fixed by e969a34bf777b7434505cb231a476b808cc32e97, transcoding/ReqdSubgroupSize.ll is moved to #12 since it's the source of the fail.
